models/yang/sonic/sonic-sflow.yang

```diff
@@ -32,9 +39,12 @@ module sonic-sflow {
                     }
 
                     leaf collector_ip {
+                        mandatory true;
                         type inet:ip-address;
+                        sonic-ext:custom-validation ValidateIp;
```

Comment:
ValidateIp is too generic. Can we change it to "ValidateSflowIP" or "ValidateCollectorIP" ?

Reply:
Sure